features/google/src/views/ministories/reportActivities.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import ReportStory from "./reportStory.jsx";
+import analysisKeys from "../../model/analyses/analysisKeys.js";
+
+class ActivitiesReport extends ReportStory {
+    constructor(props) {
+        super(props);
+        this._neededReports = [analysisKeys.reportActivities];
+    }
+
+    get title() {
+        return "Importing status";
+    }
+
+    get reportData() {
+        return this.reports[analysisKeys.reportActivities];
+    }
+
+    _renderSummary() {
+        return (
+            <>
+                <p>
+                    Google has recorded {this.reportData.length} types of
```

Comment:
No translation strings for this?

Reply:
Not yet (that's basically the reason I haven't merge it)

Reply:
Should we translate reports?

Reply:
Idk, my expectation is that we should

Reply:
We almost never translated them (we have a few translations now, but could also be good enough like that)

Reply:
I got first that no, we were not going to do it, but then yes, we would do it, at any rate this is a user-facing component, right? The user will see this, so eventually we should probably translate them.